src/couch_replicator_scheduler.erl

```diff
@@ -266,57 +260,24 @@ remove_job_int(#job{} = Job) ->
     ets:delete(?MODULE, Job#job.id).
 
 
--spec job_count() -> non_neg_integer().
-job_count() ->
-    ets:info(?MODULE, size).
-
-
 -spec running_job_count() -> non_neg_integer().
 running_job_count() ->
-    MatchSpec = [{#job{pid = '$1', _='_'},
-        [{is_pid,'$1'}],
-        [true]}
-    ],
-    ets:select_count(?MODULE, MatchSpec).
-
+    ets:info(?MODULE, size) - pending_job_count().
 
 -spec running_jobs() -> [#job{}].
 running_jobs() ->
-    MatchSpec = [{#job{pid = '$1', _='_'},
-        [{is_pid,'$1'}],
-        ['$_']}
-    ],
-    ets:select(?MODULE, MatchSpec).
+    ets:tab2list(?MODULE) -- pending_jobs().
```

Comment:
Superficially looking at it, this could be costly. First we'd make a full copy of all the jobs in tab2list. Then a copy in pending_jobs(). Then another copy is the resulting list. L1--L2 complexity is about O(length(L1)*length(L2)) as well.

Since we are returning whole objects perhaps `ets:match_object(Tab, Pattern)` would work here.